ush/machine/hera.yaml

```diff
@@ -13,6 +13,7 @@ platform:
   QUEUE_FCST: batch
   PARTITION_HPSS: service
   QUEUE_HPSS: batch
+  BATCH_RUN_CMD: srun --export=ALL
```

Comment:
I'm hoping this will be one way to reduce complexity on the batch run command since it should be the same for most of the components on a system. The flags may change from task to task, and uw yaml for driver will handle that.

Reply:
Does this command run on a compute node, or on the head node? Hopefully the former...